Utilites/RZXBlenderModelExportIndexed.py

```diff
@@ -18,134 +18,135 @@ def __init__(self):
         self.n = [0.0, 0.0, 0.0]
         self.u = [0.0, 0.0]
 
-#determine if two modal data objects are equal
-def mDataEQ(md1, md2):
-    if md1.v == md2.v and md1.n == md2.n and md1.u == md2.u:
-        return True
-    return False
+    def __hash__(self):
+        return hash(str(self.v) + "-" + str(self.n) + "-" + str(self.u))
```

Comment:
String concatenation via `+` in Python is slow, try `"-".join([str(self.v), str(self.n), str(self.u)])`